server/app/services/export/PdfExporter.java

```diff
@@ -369,6 +370,20 @@ private void renderProgramBlock(
         document.add(
             text(question.getQuestionHelpText().getDefault(), PARAGRAPH_FONT, indentationLevel));
       }
+
+      // Adds a line describing whether the question is optional or not
+      Optional<ProgramQuestionDefinition> programQuestionDefinition =
+          block.programQuestionDefinitions().stream()
+              .filter(pqd -> pqd.id() == question.getId())
+              .findFirst();
+      if (programQuestionDefinition.isPresent()) {
+        document.add(
+            text(
+                "isOptional: " + programQuestionDefinition.get().optional(),
```

Comment:
```suggestion
                programQuestionDefinition.get().optional() ? "Optional Question" : "Required Question",
```

(blocking) It will be easier for people to scan through if you put more explicit text.